content/tracing/advanced_usage/priority_sampling.md

```diff
@@ -40,9 +40,31 @@ public class MyClass {
 ```
 {{% /tab %}}
 {{% tab "Python" %}}
+To enable sampling priority in the tracer:
 ```python
-This is some python code
+tracer.configure(priority_sampling=True)
 ```
+
+To set a custom priority to a trace:
+
+```python
+from ddtrace.ext.priority import USER_REJECT, USER_KEEP
+
+context = tracer.context_provider.active()
```

Comment:
should we use `tracer.current_span` no? Let's hide the `context_provider` from our docs.